src/runtime/pay-flow-test.js

```diff
@@ -164,6 +164,47 @@ describes.realWin('PayStartFlow', {}, env => {
     await expect(flowPromise).to.eventually.be.undefined;
   });
 
+  it('should trigger the contribution flow if given contribution productType', async () => {
```

Comment:
This test won't pass because the response callback isn't getting triggered (which adds the original request data to the paymentResponse). See line 1005 of the original file for an example (the test called 'should start flow with data from request').

Reply:
Turns out it actually wasn't passing because I was passing in the product type incorrectly.

Reply:
Ahh, good to know.